lib/network/validation/proto/Makefile

```diff
@@ -8,4 +8,4 @@ all: $(IDL)
 	protoc -I=. -I=$$PROTO_INCLUDE \
 		-I=/gopath/src/github.com/gravitational/gravity/vendor/github.com/gravitational/satellite/agent/proto \
 		$^ \
-		--gofast_out=plugins=grpc,$(deps):. 
+		--gogo_out=plugins=grpc,$(deps):.
```

Comment:
`gofast` seems to generate a reference to golang protobuf package while we're [using](https://github.com/gravitational/gravity/blob/dmitri/5.5.x/4216-grpc-reproducible-build/Gopkg.toml#L74) the one from gogo repository (and so does the vendored k8s.io).